plugins/fasthttp/client/client_intercepter_test.go

```diff
@@ -63,9 +63,9 @@ func TestClientInvoke(t *testing.T) {
 	spans := core.GetReportedSpans()
 	assert.NotNil(t, spans, "spans should not be nil")
 	assert.Equal(t, 1, len(spans), "spans length should be 1")
-	assert.Equal(t, "GET:http://localhost/", spans[0].OperationName(), "operation name should be GET:/")
+	assert.Equal(t, "GET:http://localhost/", spans[0].OperationName(), "operation name should be GET:http://localhost/")
 	assert.Equal(t, "Http", spans[0].SpanLayer().String(), "SpanLayer should be Http")
-	assert.Equal(t, int32(5019), spans[0].ComponentID(), "ComponentID should be 5014")
+	assert.Equal(t, int32(5019), spans[0].ComponentID(), "ComponentID should be 5019")
```

Comment:
Looks like this component ID did [not exist in the backend](https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/component-libraries.yml). will you create a PR on the main repo?

Reply:
After merging this PR, I will submit the PR for the main repo: skywalking

Reply:
ID should be submitted and merged first.

Reply:
Otherwise, others may have registered your ID before your submission.

Reply:
Okay, I have submitted the PR on the main repo